cmd/skopeo/copy.go

```diff
@@ -65,7 +65,7 @@ func copyHandler(context *cli.Context) {
 		logrus.Fatalf("Error parsing manifest: %s", err.Error())
 	}
 	for _, layer := range layers {
-		stream, err := src.GetBlob(layer)
+		stream, _, err := src.GetBlob(layer)
```

Comment:
It would be nice to use the size here. At least add a comment, please, I need to add digest verification here anyway and that will make sure I don’t forget.